cpuinfo_linux.go

```diff
@@ -158,3 +159,40 @@ func getCPUVariant() (string, error) {
 
 	return variant, nil
 }
+
+func getAmd64MicroArchLevel() (string, error) {
+	flags, err := getCPUInfo("flags")
+	if errors.Is(err, errNotFound) {
+		return "", fmt.Errorf("failure getting CPU flags: %v", err)
+	}
+
+	containsAll := func(set map[string]interface{}, toMatch []string) bool {
+		for _, m := range toMatch {
+			if _, ok := set[m]; !ok {
+				return false
+			}
+		}
+		return true
+	}
+
+	flagSet := map[string]interface{}{}
+	for _, flag := range strings.Split(flags, " ") {
+		flagSet[flag] = true
+	}
+
+	// https://unix.stackexchange.com/questions/631217/how-do-i-check-if-my-cpu-supports-x86-64-v2
+	level := 1
+	if containsAll(flagSet, []string{"lm", "cmov", "cx8", "fpu", "fxsr", "mmx", "syscall", "sse2"}) {
+		level = 1
+	}
+	if level == 1 && containsAll(flagSet, []string{"cx16", "lahf_lm", "popcnt", "sse4_1", "sse4_2", "ssse3"}) {
```

Comment:
```suggestion
	if containsAll(flagSet, []string{"cx16", "lahf_lm", "popcnt", "pni", "sse4_1", "sse4_2", "ssse3"}) {
```

Looks like "SSE3" is missing and its cpu flag seems to be `pni`

> GOAMD64=v2: all v1 instructions, plus CMPXCHG16B, LAHF, SAHF, POPCNT, SSE3, SSE4.1, SSE4.2, SSSE3.

Reply:
Added to the flag.
Thanks for catching it.